app/views/staff_dashboard/index.html.erb

```diff
@@ -34,40 +34,32 @@
       <div class="container text-center mx-auto">
         <%= link_to 'Sign Out All Users', staff_dashboard_sign_out_all_users_path, class: 'btn btn-secondary mx-auto text-center' %>
       </div>
-      <table class="staff-users-table text-center mb-5 table table-striped" >          
+      <table class="staff-users-table text-center mb-5 table table-striped table-sort table-arrows">
         <thead class="table-primary">
           <tr>
-            <th class="name-header">Name
-              <span onclick="sortTable('staff-users-table', 0)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="email-header">Email
-              <span onclick="sortTable('staff-users-table', 1)"><%= fa_icon('arrow-down') %></span></th>
+            <th class="name-header">Name</th>
+            <th class="email-header">Email</th>
             <th class="name-header">Flagged?</th>
-            <th class="certifications-header">Certifications in <%= @space.name %>
-              <span onclick="sortTable('staff-users-table', 2)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="email-header"> # of visits <br> (Past 2
-              months)<span onclick="sortTable('staff-users-table', 2)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="last-seen-header">Last Seen
-              <span onclick="sortTable('staff-recent-users-table', 2)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="action-header">Sign Out of <%= @space.name.capitalize %></th>
+            <th class="certifications-header">Certifications in <%= @space.name %></th>
+            <th class="email-header"> # of visits <br> (Past 2 months)</th>
+            <th class="last-seen-header">Last Seen</th>
+            <th class="action-header disable-sort">Sign Out of <%= @space.name.capitalize %></th>
```

Comment:
I hate the look of .capitalize, I think all spaces currently have their correct capitalisation in .name

Reply:
True